src/store/metadata/oxigraph/store.py

```diff
@@ -143,13 +143,55 @@ def get_topics(self) -> Optional[Dict]:  # pragma: no cover
         """
         Get all topics
         """
-        raise NotImplementedError
+        graph = self.db
+
+        result: Graph = construct_dataset_topics(graph)
+
+        # Serialize the graph into jsonld
+        data = json.loads(result.serialize(format="json-ld"))
 
-    def get_topic(self) -> Optional[Dict]:  # pragma: no cover
+        # Use a context file to shape our jsonld, removing long form references
+        data = jsonld.flatten(
+            data, {"@context": constants.CONTEXT, "@type": "hydra:Collection"}
+        )
+
+        for idx, topic in enumerate(data["@graph"][0]["topics"]):
+            topic_id = topic["@id"].split("/")[-1]
+            data["@graph"][0]["topics"][idx] = self.get_topic(topic_id)
+
+        data["@graph"][0]["@context"] = "https://staging.idpd.uk/#ns"
```

Comment:
a good question RE context url, this is one of those "for now" things.

down the line any explicit mention of staging needs to be replaced with (I'd imagine) the contents of an environment variable so we can deploy our applications in other enivronments (production.idpd.uk, sandbox.idpd.uk, etc etc).

but given how much its hardcoded in at the moment lets keep it simple - this is fine and we'll decouple it later along with all the others.